server/controllers/metadata.controller.js

```diff
@@ -173,7 +175,13 @@ function vote(delta) {
   }
 }
 
-function updateSingle(req, res, next, fromRevision = false) {
+function updateSinglePromise(req, res, next, fromRevision = false) {
+  return new Promise((resolve) => {
+    updateSingle(req, res, next, fromRevision, resolve)
+  })
+}
+
+function updateSingle(req, res, next, fromRevision = false, resolve) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Move parameter "fromRevision" after parameters without default value. ([javascript:S1788](https://sonarcloud.io/organizations/aumanjoa-github/rules#rule_key=javascript%3AS1788))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=chronas-api&pullRequest=29&issues=AWbY54tdSgN4uCclP4ph&open=AWbY54tdSgN4uCclP4ph)

[ISSUE_KEY]: AWbY54tdSgN4uCclP4ph